site/app/templates/UserProfile.twig

```diff
@@ -132,9 +132,36 @@
                     </div>
                 </div>
             </div>
+            <div>
+            <button class="addpwa btn btn-primary">Install App</button>
+            </div>
         </div>
     </div>
 </div>
+<script>
+        let storedPrompt;
+        let pwaBtn = $('.addpwa');
+        pwaBtn[0].style.display = 'none';
+        window.addEventListener('beforeinstallprompt', (e) => {
+       
+        e.preventDefault();
+
+        storedPrompt = e;
+        pwaBtn[0].style.display = 'block';
+        pwaBtn[0].addEventListener('click', (e) => {
+         pwaBtn[0].style.display = 'none';
+         storedPrompt.prompt();
+         storedPrompt.userChoice.then((choiceResult) => {
+         if (choiceResult.outcome === 'accepted') {
```

Comment:
Is this if statement required? I don't think this action would need to be logged but it doesn't seem clear what this logic is doing

Reply:
The upper two lines hides the `add pwa` button and the if statement just logs the successful installation of the pwa in the console. The log in the console could be removed if suggested

Reply:
I was referring to the `console.log` in the if/else statement. I don't think that is necessary since the user will most likely never see that and they should see visual feedback from the browser when they either accept or deny the prompt to install the PWA